core/src/main/java/com/linecorp/armeria/common/stream/DeferredStreamMessage.java

```diff
@@ -141,8 +169,8 @@ public final void close() {
      */
     public final void close(Throwable cause) {
         requireNonNull(cause, "cause");
-        final DefaultStreamMessage<T> m = new DefaultStreamMessage<>();
-        m.close(cause);
+        final StreamMessage<T> m = StreamMessage.of();
+        m.abort(cause);
```

Comment:
How about adding `StreamMessage.aborted(Throwable)` ?

Reply:
As like `Flux.error(Throwable)`? Sounds good to me. 👍